### PR TITLE
Sequential metadata decryption for better progress info

### PR DESCRIPTION
**Pull request by agix**

### CODE REVIEW

File: src/lib/utils.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ export function escapeRegExp(s) {
 }
 
 export function defaultProgress(status) {
-  const seconds = Math.trunc(Date.now() / 1000);
+  const seconds = Math.trunc(Date.now());
```
Comment (Calyhre) [inline src/lib/utils.js:128]: So it's not seconds anymore?

**Review by Calyhre — APPROVED** [review]

File: src/lib/utils.js (hunk 1 above)
Comment (agix) [reply src/lib/utils.js:128]: It was better to compare between parallelized and sequential metadata decryption.

Comment (Calyhre) [reply src/lib/utils.js:128]: But why is this var named `seconds` if it's not? 🍝

Comment (agix) [reply src/lib/utils.js:128]: Because I'm stupid

**Pull request merged**